sdk/core/azure-core-amqp/src/main/java/module-info.java

```diff
@@ -3,18 +3,23 @@
 
 module com.azure.core.amqp {
     requires transitive com.azure.core;
-    requires transitive reactor.core;
 
-    requires com.microsoft.azure.qpid.protonj.extensions;
-    requires proton.j;
-    requires org.reactivestreams;
+    requires transitive com.microsoft.azure.qpid.protonj.extensions;
```

Comment:
is there a reason we're adding transitive here?

Reply:
`transitive` is  for implicit readability,  so in service I do not need to depend on it and it will be available implecitly.